processor/configuration/src/main/resources/schemas/actions/aws_lambda_sink_0.1.json

```diff
@@ -0,0 +1,92 @@
+{
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "aws_function",
+    "aws_region",
+    "aws_access_key",
+    "aws_secret_key"
+  ],
+  "properties": {
+    "aws_function": {
+      "title": "Function Name",
+      "description": "The Lambda Function name",
+      "type": "string"
+    },
+    "aws_access_key": {
+      "title": "Access Key",
+      "x-group": "credentials",
+      "oneOf": [
+        {
+          "title": "Access Key",
+          "description": "The access key obtained from AWS",
+          "type": "string",
+          "format": "password"
+        },
+        {
+          "description": "An opaque reference to the aws_access_key",
+          "type": "object",
+          "properties": {}
+        }
+      ]
+    },
+    "aws_secret_key": {
+      "title": "Secret Key",
+      "x-group": "credentials",
+      "oneOf": [
```

Comment:
@kelvah 
FYI connector uses this `oneOf` entry for passwords/secrets because when you create the action the password is sent but when you retrieve it, it is masked.
I don't know if this is a problem for the form generation, if so, you can check with Riccardo because they had to solve the same issue already

Reply:
Thanks for the heads up @danielezonca. I'll look into it.